bin/sozo/tests/register_test.rs

```diff
@@ -0,0 +1,54 @@
+mod utils;
+
+use dojo_test_utils::compiler::build_test_config;
+use dojo_test_utils::migration::prepare_migration;
+use dojo_test_utils::sequencer::{
+    get_default_test_starknet_config, SequencerConfig, TestSequencer,
+};
+use scarb::ops;
+use sozo::ops::migration::execute_strategy;
+use starknet::accounts::Account;
+use starknet::core::types::{BlockId, BlockTag};
+use utils::snapbox::get_snapbox;
+
+#[tokio::test(flavor = "multi_thread")]
+async fn reregister_models() {
+    let config = build_test_config("../../examples/spawn-and-move/Scarb.toml").unwrap();
+    let ws = ops::read_workspace(config.manifest_path(), &config)
+        .unwrap_or_else(|op| panic!("Error building workspace: {op:?}"));
+
+    let base_dir = "../../examples/spawn-and-move";
+    let target_dir = format!("{}/target/dev", base_dir);
+    let migration = prepare_migration(base_dir.into(), target_dir.into()).unwrap();
+
+    let sequencer =
+        TestSequencer::start(SequencerConfig::default(), get_default_test_starknet_config()).await;
+
+    let mut account = sequencer.account();
+    account.set_block_id(BlockId::Tag(BlockTag::Pending));
+
+    execute_strategy(&ws, &migration, &account, None).await.unwrap();
+    let world_address = &format!("0x{:x}", &migration.world_address().unwrap());
+    let account_address = &format!("0x{:x}", account.address());
+    let private_key = &format!("0x{:x}", sequencer.raw_account().private_key);
+    let rpc_url = &sequencer.url().to_string();
+
+    let moves_model_class_hash =
+        "0x511fbd833938f5c4b743eea1e67605a125d7ff60e8a09e8dc227ad2fb59ca54";
```

Comment:
We may want to fetch the `Moves` from the world, to ensure this test remains valid even if some classes change in the future.

Reply:
Yes, this is a really good point, thank you!